packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -2040,10 +2040,10 @@ type MessagesType = {
   'scenes.preferences.loading.email.description3': 'and to send bitcoin payment alerts when you receive funds.'
   'scenes.preferences.loading.email.title': 'Email address'
   'scenes.priceindexseries.timefilters.all': 'All'
-  'scenes.priceindexseries.timefilters.day': 'Day'
-  'scenes.priceindexseries.timefilters.month': 'Month'
-  'scenes.priceindexseries.timefilters.week': 'Week'
-  'scenes.priceindexseries.timefilters.year': 'Year'
+  'scenes.priceindexseries.timefilters.day': '1D'
+  'scenes.priceindexseries.timefilters.month': '1M'
+  'scenes.priceindexseries.timefilters.week': '1W'
+  'scenes.priceindexseries.timefilters.year': '1Y'
```

Comment:
can we move these to generic copies since it will be used in other places for sure?
e.g.
`copy.all: 'All'`
`copy.1m: '1m'`

Reply:
Do we even need to translate 1d?